zookeeper-api/src/test/java/org/apache/helix/zookeeper/impl/TestZooKeeperConnection.java

```diff
@@ -53,10 +53,11 @@ void testPersistWatcher() throws Exception {
       _zk.writeData(path, ("datat"+i).getBytes(), -1);
       _zk.create(path+"/c2_" +i, null, ZooDefs.Ids.OPEN_ACL_UNSAFE, CreateMode.PERSISTENT);
     }
+    System.out.println("testPersistWatcher: rafter register one time listener, original listener received event count: " + get_count[0]);
```

Comment:
nit: after 
This is old test case, but we decided to use logger instead of system.  we can do it as part of other change.